sdks/python/apache_beam/runners/direct/watermark_manager.py

```diff
@@ -128,8 +129,9 @@ def _update_pending(self, input_committed_bundle, applied_ptransform,
     if input_committed_bundle and input_committed_bundle.has_elements():
       completed_tw.remove_pending(input_committed_bundle)
 
-  def _refresh_watermarks(self, applied_ptransform):
+  def _refresh_watermarks(self, applied_ptransform, side_inputs_container):
```

Comment:
Similar to the above comment. Can you use self._side_inputs_container instead of adding this new argument here?

Reply:
You're right--Maria, can you fix this?

Reply:
Done